Explain a bit more about Prioritization summaries (#452)
rust-lang/rust-forge: src/compiler/prioritization/procedure.md

### Add details to the Agenda

> In order to summaries issues take into account that is important to add

Reviewer: ```suggestion
In order to summarize issues take into account that it is important to add
```

> details that make the meeting be more effective and faster. For that

Reviewer: ```suggestion
details that will speed up the meeting. For that
```

> consider mentioning important actors, important details like is a

Reviewer: ```suggestion
consider mentioning important actors, important details like if it is a
```

> regression and when it has regressed or if it already has a PR.

Reviewer: ```suggestion
regression and when it has regressed or if it already has a PR attached to it.
```

Properly read the issue to summarize it and also process the
Prioritization WG stream topic related to the issue to extract other

> useful information.

Reviewer 2: We also summarize beta/stable nominations which have PRs and as such don't get discussed. Not sure how to make it more clear in the text here though

Author: Yeah agree, maybe writing in a more general way would be better. I guess we should merge this and keep improving the text in separate PRs.
